kudo-e2e-test.yaml.tmpl

```diff
@@ -3,7 +3,7 @@ kind: TestSuite
 testDirs:
 - ./test/e2e
 commands:
-- command: ./bin/kubectl-kudo init --unsafe-self-signed-webhook-ca --kudo-image kudobuilder/controller:%version% --kudo-image-pull-policy Never
+- command: ./bin/kubectl-kudo init --wait --unsafe-self-signed-webhook-ca --kudo-image kudobuilder/controller:%version% --kudo-image-pull-policy Never
```

Comment:
This is the only actual change.

All other changes are:
* Remove the 00-assert.yaml from all e2e-tests (which ensured that KUDO manager was running)
* Move all other test files one number up ( which git didn't recognise correctly, therefore the many diffs :( )